app/lib/champaign_queue.rb

```diff
@@ -7,7 +7,7 @@ module ChampaignQueue
   extend self
 
   def push(payload, opts = {})
-    if Rails.env.production? || Settings.publish_champaign_events
+    if Rails.env.production? && Settings.publish_to_queue
```

Comment:
why didn't like the other version?

I like the `event` term more for this, since we're not pushing to a queue necessarily, events can also be simple HTTP requests. We can leave out champaign and maybe leave it as `publish_events` if you prefer